### ISSUE TITLE
Error: No such command "home"

### ISSUE THREAD

**Issue by DerAndere1** [Jason2866/Portable_VSC_PlatformIO#4]
With the last release, when I hit Strg + Shift + P to open the command pallette and run command "PlatformIO: Home",  I get the following error:
```
Error: Usage: __main__.py [OPTIONS] COMMAND [ARGS]...
Try '__main__.py -h' for help.

Error: No such command "home"

	at c:\Users\cb\Downloads\VSC_PlatformIO_Python391\Vsc.Pyt_39\t\vscode\data\extensions\platformio.platformio-ide-2.3.2\node_modules\platformio-node-helpers\dist\index.js:1:5780
	at ChildProcess.a (c:\Users\cb\Downloads\VSC_PlatformIO_Python391\Vsc.Pyt_39\t\vscode\data\extensions\platformio.platformio-ide-2.3.2\node_modules\platformio-node-helpers\dist\index.js:1:931270)
	at ChildProcess.emit (events.js:315:20)
	at ChildProcess.EventEmitter.emit (domain.js:467:12)
	at ChildProcess.cp.emit (c:\Users\cb\Downloads\VSC_PlatformIO_Python391\Vsc.Pyt_39\t\vscode\data\extensions\pl...

```

**Comment by DerAndere1** [comment]
A workaround is to press Ctrl + Shift + P to open the command pallette and run command "PlatformIO: Upgrade PlatformIO Core ". Alternatively, run the command `pio upgrade` in the terminal. 

A proper fix would be to publish a new release of Portable_VSC_PlatformIO that comes bundled with PlatformIO core 6

**Comment by Jason2866** [comment]
Oh, it did still worked! Didnt expected. I moved away from windows for over a year and didnt touch this for a long while.
I will have a look, if my old Windows PC still boots.

**Comment by DerAndere1** [comment]
Can you describe how to compile the source code to get 
VSC_PlatformIO_Python391.zip ? Then others/I could help you to maintain and update this software.

**Comment by Jason2866** [comment]
Unpack, take a look in `Lib\site-packages` there you find two folders `platformio*.*` 
Delete and replace with the ones you have in your updated version. Zip the package.
Should work (if i remember correctly)

**Comment by DerAndere1** [comment]
I had to replace all contents of `Lib\site-packages`. An easier solution is:
1.  Download file VSC_PlatformIO_Python391.zip from https://github.com/Jason2866/Portable_VSC_PlatformIO/releases
2. Unzip VSC_PlatformIO_Python391.zip
3. In the unzipped VSC_PlatformIO_Python391 folder, select and run the executable "VS code.exe"
4. Click in the terminal of VS code and run the command `pio upgrade` and Wait for succesful upgrade. This updates the contents of VSC_PlatformIO_Python391
5. Close VS code
6. Create a new zip archive from the updated VSC_PlatformIO_Python391 folder

**Comment by Jason2866** [comment]
Have you checked, if it is still portable? Tried on a different drive/folder?

**Comment by DerAndere1** [comment]
Yes on same computer. but I have to test on a different Windows computer.

**Comment by Jason2866** [comment]
Try latest release 1.6. Should work builded in a Windows11 ARM VM

**Comment by Jason2866** [comment]
> Click in the terminal of VS code and run the command pio upgrade and Wait for succesful upgrade. This updates the contents of VSC_PlatformIO_Python391

This brakes portable install of Platformio. The compiled python *.exe files needs patching (removing the path before `python.exe`)

**Comment by DerAndere1** [comment]
Thanks a lot for the quick action. Seems to be fixed with release 1.6. I am closing this issue for now.
